src/Breadcrumb/README.md

```diff
@@ -42,10 +44,29 @@ notes: |
 ```jsx live
 <Breadcrumb
   links={[
-    { label: 'Link 1', url: '/link-1' },
-    { label: 'Link 2', url: '/link-2' },
-    { label: 'Link 3', url: '/link-3' },
+    { label: 'Link 1', url: '#link-1' },
+    { label: 'Link 2', url: '#link-2' },
+    { label: 'Link 3', url: '#link-3' },
   ]}
   spacer={<span className="custom-spacer">/</span>}
 />
 ```
+
+### Theme variables (SCSS)
+
+```scss
```

Comment:
The theme variables are no longer necessary in the README with the [changes recently merged](https://github.com/edx/paragon/pull/931). Just a symptom of a delay in reviewing this PR, my apologies.